templates/necto/project_templates/0_mikrosdk2/CMakeLists.txt

```diff
@@ -19,14 +18,14 @@ add_executable(${PROJECT_NAME} main.c)
 ############################ ${PROJECT_NAME} GENERATED CODE START ###########################
 ############################ ${PROJECT_NAME} GENERATED CODE END ###########################
 
-find_package(MikroC.Core REQUIRED)
+find_package_me(MikroC.Core REQUIRED)
```

Comment:
Lets remove all checks for `MikroC.Core` and `MikroC.System` as these 2 can not fail.

`MikroC.Core` is a prerequisite and `MikroC.System` is present only for mikroC legacy setups (also a prerequisite there).

It is full-proof from any kind of fail i believe.

Reply:
Removed checking and tested them